main.tf

```diff
@@ -13,6 +13,39 @@ module "label" {
   tags       = "${var.tags}"
 }
 
+resource "aws_s3_bucket" "cache_bucket" {
+  bucket_prefix = "${local.b_name_hyphen}"
```

Comment:
Oh, can you explain why we should use `bucket_prefix` instead of `bucket`?

Reply:
I want to guarantee the bucket can be created.
An issue I have is project names can be common across clients, and when the bucket can't be created it takes a junior person a while to figure out that it is because the dynamically generated bucket name from the `terraform-null-label` has an overlapping s3 bucket from some other company in the world.

Reply:
ok, no strong opinion. @aknysh thoughts?

Reply:
See opinion: https://github.com/cloudposse/terraform-aws-codebuild/pull/16#discussion_r186841056